dataduct/database/tests/test_column.py

```diff
@@ -0,0 +1,103 @@
+"""Tests for the Column class
+"""
+from unittest import TestCase
+from nose.tools import eq_
+from nose.tools import raises
+
+from ..column import Column
+
+
+class TestColumn(TestCase):
+    """Tests for the Column class
+    """
+    @staticmethod
+    def test_column_construction():
+        """Tests that the constructor for the column class is working
+        """
+        column = Column(
```

Comment:
shouldn't we be testing the parser function as most of the tests here seem to just check the default values .. or may be my brain isn't working anymore.